.ci/eclipse-compiler-javac.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+set -e
+
+ECJ_VERSION="4.6.1"
+mkdir -p target/classes
+mkdir -p target/eclipse
+
+if [ ! -e ~/.m2/repository/"ecj-$ECJ_VERSION.jar" ]; then
+    wget http://central.maven.org/maven2/org/eclipse/jdt/core/compiler/ecj/$ECJ_VERSION/ecj-$ECJ_VERSION.jar -P ~/.m2/repository/
+fi
+
+java -jar ~/.m2/repository/ecj-$ECJ_VERSION.jar -target 1.8 -source 1.8 -cp $1 \
+        -nowarn:[./target/generated-sources/antlr] -d target/eclipse-compile \
```

Comment:
@romani "-err" is used behind the scene in the property file. All inspections are enabled as `error` or `ignore` (those ones, that produce some errors). `error` in property file is the same as `err` here.